primers/pcr/pcr.go

```diff
@@ -0,0 +1,138 @@
+package pcr
+
+import (
+	"github.com/TimothyStiles/poly/primers"
+	"github.com/TimothyStiles/poly/transform"
+	"index/suffixarray"
+	"sort"
+	"strings"
+)
+
+// https://doi.org/10.1089/dna.1994.13.75
+var minimalPrimerLength int = 15
+
+func DesignPrimersWithOverhangs(sequence, forwardOverhang, reverseOverhang string, targetTm float64) (string, string) {
```

Comment:
Exported functions need a comment above function definition describing its use that started with the function name. i.e `// DesignPrimersWithOverhangs takes <talk about inputs and outputs, and any quirks here>`.